### PR TITLE
Fix the kubernetes logging issue.

### PR DESCRIPTION
**Pull request by abhijitherekar**
#### Description
*Changes proposed in this pull request.*

#### Test plan
*Outline the test plan used to test this change before merging it.*

> Fixes #.

### CODE REVIEW

File: controller/internal/processmon/processmon.go [hunk 1]
```diff
@@ -214,7 +217,7 @@ func (p *RemoteMonitor) LaunchRemoteEnforcer(
 		refNSPath,
 	)
 	cmd.Env = append(os.Environ(), newEnvVars...)
-
+	fmt.Println(" \n\n *** ABHI: The env for the remote enforcer are: ", cmd.Env, "\n\n\n ")
```
Comment (mheese) [inline controller/internal/processmon/processmon.go]: this guy has to go :)

File: controller/pkg/env/parameters.go [hunk 2]
```diff
@@ -44,6 +45,6 @@ func GetParameters() (logToConsole bool, logID string, logLevel string, logForma
 			compressedTagsVersion = claimsheader.CompressionTypeV2
 		}
 	}
-
+	DisableLogWrite, _ = strconv.ParseBool(os.Getenv(constants.EnvDisableLogWrite))
```
Comment (mheese) [inline controller/pkg/env/parameters.go:49]: add a comment why it is okay to skip the error checking

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "K8s_logging_fix",
    "component": "enforcerd",
    "pr-id": "1356",
    "commit-sha": "e4f9643a4ac6b0c3d9c8b58f06033a34fd25c491"
  },
  {
    "project": "K8s_logging_fix",
    "component": "trireme-lib",
    "pr-id": "860",
    "commit-sha": "32f5af7359bf99a478017120f207bbd35481c385"
  }
]
```

**Review by mheese — APPROVED** [review]
lgtm! 👍 

I still don't really like the new flag, but it's not the end of the world. If viper is not helping us, let's do it this way for now.

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "K8s_logging_fix",
    "component": "enforcerd",
    "pr-id": "1356",
    "commit-sha": "ea3996fdca046cadc070ecd0998efb696857b6a6"
  },
  {
    "project": "K8s_logging_fix",
    "component": "trireme-lib",
    "pr-id": "860",
    "commit-sha": "32f5af7359bf99a478017120f207bbd35481c385"
  }
]
```

**Pull request merged**